src/ClientInfo.h

```diff
@@ -78,6 +78,8 @@ class ClientInfo : public hash_link
     unsigned int quotaPeekReserv() const; ///< returns the next reserv. to pop
     void quotaDequeue(); ///< pops queue head from queue
     void kickQuotaQueue(); ///< schedule commHandleWriteHelper call
+    /// selects the delayed head descriptor for writing or quotaDequeue()
+    bool writeOrQuotaDequeue();
```

Comment:
This class methods are named _inconsistently_, but AFAICT, there is no other queue in this class, and "quota" is mostly used as a method name _prefix_ to group/mark quota-related methods. I do not think we should be using "quota" in the middle of this new method name.

```suggestion
    void writeOrDequeue();
```

Reply:
Done.